PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | [yes|no]
+| New feature?  | [yes|no]
+| BC breaks?    | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass?   | [yes|no]
+| Fixed tickets | [comma separated list of tickets fixed by the PR]
```

Comment:
maybe we should add "if any" here as well. I've seen alot of people creating issues just to reference them in the PR. Or they self-reference the PR itself.